doc/source/tune/api_docs/stoppers.rst

```diff
@@ -0,0 +1,55 @@
+.. _tune-stoppers:
```

Comment:
Can you be sure to reference this in the `user-guide` too, under the existing Stopping Trials section?

And maybe also have a backward reference to that section, since it mentions a couple different ways to provide stoppers to Tune (dict, function, class)